website/content/api-docs/acl/index.mdx

```diff
@@ -201,6 +201,7 @@ $ curl \
 - `LastErrorMessage` - The last error message produced at the time of `LastError`.
   An empty string indicates that no sync has resulted in an error.
 
+<<<<<<< HEAD
```

Comment:
Not sure if this is intentional.. looks like an in-progress git merge. There are some other ones below as well.

Reply:
bad patch artifact - apologies for sloppiness

Reply:
No worries!